f2/src/summary/WhatWasAffectedSummary.js

```diff
@@ -52,9 +52,15 @@ export const WhatWasAffectedSummary = (props) => {
           <H2 fontWeight="normal">
             <Trans id="confirmationPage.ImpactTitle" />
           </H2>
+          {impact.edited && (
+            <H3 fontWeight="normal" paddingLeft="1rem">
+              <Trans id="confirmationPage.editedTag" />
+            </H3>
+          )}
           <EditButton
-            path="/whatwasaffected"
             label="confirmationPage.ImpactTitle.edit"
+            path="/whatwasaffected"
+            edited={impact.edited}
           />
         </Flex>
         {containsData(impact) ? (
```

Comment:
I know containsData is overly complex. But for string operation, it makes sense to have a check that the string is not empty.
str is not undefined, is not null, size is not zero and doesn't just contain white spaces